transitions/index.js

```diff
@@ -125,7 +125,9 @@ const loadTransitions = (autoEnableSystemTransitions = true) => {
       return logger.warn(`transition ${transition} is disabled`);
     }
 
-    self._loadTransition(transition);
+    // Needed to support tests mocking out _loadTransition, and _loadTransition
+    // not being available in prod.
+    (self._loadTransition || self.loadTransition)(transition);
```

Comment:
Damn ugly, maybe we should go back to the old way? Where `_` means it's private use with caution.